### DISCUSSION TITLE
GitHub - File view using 3.5G on Safari!

### DISCUSSION THREAD

**Discussion by gmpassos** [community/community#111141] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

The link below uses 3.5GB of RAM in Safari (while loading the page):
https://github.com/dart-lang/sdk/blob/main/CHANGELOG.md

This is likely due to JavaScript compilation.

---

Safari:
Version 15.5 (17613.2.7.1.8)

macOs:
Version 12.4 (Build 21F79)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @gmpassos ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
